src/services/class.service.ts

```diff
@@ -56,36 +56,37 @@ export class ClassService {
   }
 
   public getSimilarClasses(
-    message: types.IMessage,
+    interaction: discord.CommandInteraction,
     invalidClasses: string[],
     action: 'register' | 'unregister'
-  ): types.IEmbedData[] {
+  ): types.IInteractionEmbedData[] {
     return invalidClasses.map((invalidClass: string) => {
-      const emojiData: types.IEmojiTable[] = [];
+      const emojiData: types.IButtonLabelTable[] = [];
       const embeddedMessage: discord.MessageEmbed = new discord.MessageEmbed();
 
       embeddedMessage.setColor('#0099ff').setTitle(`${invalidClass} Not Found`);
       const shouldShowAuthorOnRegister = true;
       if (shouldShowAuthorOnRegister) {
-        embeddedMessage.setAuthor(this._messageService.getEmbedAuthorData(message));
+        embeddedMessage.setAuthor(this._messageService.getEmbedAuthorData(interaction));
       }
 
       const [similarClassID] = this.findSimilarClasses(invalidClass);
 
       // TODO check if similarity is close then decide whether to return the guess or tell them to get a mod.
 
-      embeddedMessage.setDescription(
-        `Did you mean \`${similarClassID}\`?\n` +
-          `React with ✅ to ${action} for this class.\n` +
-          'React with ❎ to close this offering.'
-      );
+      embeddedMessage.setDescription(`Did you mean \`${similarClassID}\`?\n`);
 
       // EmojiData acts as a key.
       emojiData.push({
```

Comment:
can you demo what this looks like?

Reply:
Here's the demo:
https://github.com/cs-discord-at-ucf/lion/assets/77477100/aa73a55b-5708-4591-99dc-8c0368c2445d